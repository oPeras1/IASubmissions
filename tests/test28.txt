1	1	2	2	3	3	3	3	3	4	4	5	5	5	6	6	7	7	7	8
1	1	2	3	3	9	9	10	10	10	4	5	5	5	6	6	7	8	7	8
1	1	2	2	11	11	9	9	10	4	4	4	12	12	12	6	6	8	8	8
1	13	11	11	11	11	9	14	14	14	14	15	12	12	12	6	6	6	16	17
13	13	18	11	11	9	9	9	14	14	15	15	15	12	12	19	19	16	16	17
13	13	18	20	20	20	20	20	21	22	22	22	23	23	19	19	16	16	16	17
13	18	18	18	20	20	21	21	21	21	22	22	22	23	19	24	24	24	16	17
13	25	25	26	26	27	27	21	28	21	22	29	23	23	30	24	31	24	16	17
25	25	25	26	27	27	27	28	28	21	22	29	29	23	30	30	31	24	24	17
25	32	26	26	26	26	26	33	28	29	29	29	29	34	30	35	31	31	31	36
37	32	26	38	38	26	26	33	33	33	33	34	34	34	34	35	35	35	35	36
37	32	32	32	38	38	39	33	39	40	40	41	41	41	34	34	35	35	36	36
37	37	32	32	42	39	39	39	39	39	40	40	41	34	34	43	44	36	36	45
46	37	37	42	42	42	47	47	47	39	40	34	34	34	34	43	44	36	36	45
46	37	48	48	42	42	47	47	47	39	40	49	34	34	43	43	44	44	36	45
46	37	37	48	48	50	50	51	51	40	40	49	43	43	43	44	44	52	36	45
46	48	48	48	50	50	51	51	51	51	40	49	43	43	53	44	44	52	36	45
46	54	48	54	51	51	51	55	55	56	40	49	43	53	53	57	44	52	36	58
46	54	48	54	54	55	55	55	55	56	40	49	43	53	53	57	52	52	52	58
54	54	54	54	54	56	56	56	56	56	40	43	43	43	53	57	57	57	58	58