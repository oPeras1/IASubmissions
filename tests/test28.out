T	1	L	L	I	I	I	I	3	4	L	T	T	T	S	6	L	L	L	8
T	T	L	3	3	9	9	T	T	T	L	5	T	5	S	S	L	8	7	L
T	1	L	2	S	S	L	9	T	4	L	L	S	S	12	S	6	L	L	L
1	S	11	S	S	11	L	S	S	14	14	T	12	S	S	6	6	6	T	17
S	S	L	11	11	L	L	9	S	S	T	T	T	12	12	S	S	T	T	17
S	13	L	T	T	T	20	20	L	22	I	22	L	L	S	S	16	16	T	I
13	L	L	18	T	20	L	L	L	21	I	22	22	L	19	L	L	L	16	I
13	S	25	L	26	S	S	21	T	21	I	29	23	L	T	24	31	L	16	I
S	S	25	L	S	S	27	T	T	21	I	29	L	23	T	T	L	24	24	I
S	32	L	L	26	26	26	33	T	29	L	L	L	34	T	35	L	L	L	36
37	S	26	S	S	26	26	L	L	L	33	34	34	34	I	S	S	35	35	36
37	S	S	32	S	S	39	L	39	40	I	T	T	T	I	34	S	S	L	L
37	37	S	32	42	I	I	I	I	39	I	40	T	34	I	43	L	36	L	45
46	L	L	T	T	T	47	47	L	39	I	34	34	34	I	43	L	36	L	I
46	L	48	48	T	42	L	L	L	39	I	49	34	34	L	43	L	L	36	I
I	L	37	48	48	S	S	51	T	40	40	I	L	L	L	44	44	I	36	I
I	48	L	L	S	S	51	T	T	T	40	I	43	43	S	44	44	I	36	I
I	54	L	54	51	51	51	L	55	L	40	I	43	S	S	L	44	I	36	L
I	54	L	54	54	L	L	L	55	L	40	I	43	S	53	L	52	I	52	L
54	I	I	I	I	56	56	56	L	L	40	43	43	43	53	L	L	57	L	L